src/store/modules/app.ts

```diff
@@ -53,8 +54,12 @@ export const appStore = defineStore({
     pageTitle: "",
     version: "",
     isMobile: false,
+    reloadTimer: null,
   }),
   getters: {
+    _reloadTimer(): any {
```

Comment:
I don't think we need this here.

Reply:
Thanks for your reminding.It was removed.